.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# For more details about how to release follow https://github.com/newrelic/coreint-automation/blob/main/docs/release_runbook.md
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * 1"
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+    secrets:
```

Comment:
Currently we are not consistently sending a warn message whenever a release is failing.

Checking https://github.com/newrelic/nri-postgresql/blob/master/.github/workflows/prerelease.yml we do not do the same in the different steps and we do not always include 
```
❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed.
```
Therefore, if the pre-release fails when running tests the only person notified is who created the preselease (the bot) and who modified the pipeline.


Is it the right approach to rely on such a message? Should we find something else?

Reply:
(I am adding this here to open a discussion, but it is kind of un-related to the PR, we can move it to a different place)

Reply:
I was referring to an other issue, but thanks also that is needed! 😄

Reply:
Right! I mixed them up! 😅